composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": ">= 7.4"
+        "php": "^7.4 || ^8.0"
```

Comment:
Why this change ?

See: https://github.com/symfony/symfony/pull/36876

Reply:
Lots of heated discussions in that PR 😄 .

My main rationale is the one here: https://github.com/symfony/symfony/pull/36876#issuecomment-631351947. I don't see any good arguments against that comment in the thread. Basically we don't know for example how this library will perform in PHP 8.1, so if we use >= 7.4 then a user installing it on 8.1 might have a bad experience if they find that it's not working.

That being said I don't feel too strongly about this, because there's an argument to be made that users might report issues so you can thus fix them earlier.